decidim-generators/spec/runtime/full_app_generator_spec.rb

```diff
@@ -21,7 +21,7 @@ module Decidim
         context "when running the db commands consecutively" do
           let(:command) { "decidim #{test_app} --demo --path #{repo_root}" }
 
-          let(:subcommand) { "bundle exec rails db:drop db:create db:migrate assets:precompile db:seed" }
+          let(:subcommand) { "bundle exec rails db:drop db:create db:migrate assets:precompile; bundle exec rails db:seed" }
```

Comment:
We just need to make sure `PaperTrail::Version` is being reloaded to avoid errors